pontoon/teams/views.py

```diff
@@ -77,6 +78,25 @@ def ajax_info(request, locale):
     })
 
 
+@require_POST
+@permission_required_or_403('base.can_manage_locale', (Locale, 'code', 'locale'))
+@transaction.atomic
+def ajax_update_info(request, locale):
+    description = request.POST.get('team_info', None)
+    tags = bleach.ALLOWED_TAGS[:]
+    # allow <p>
```

Comment:
Start comments with the capital letter. We also allow `<br>`.

I'd add blank lines above the `tags =` and `attrs =` lines.